apps/v3_api/test/headers_test.exs

```diff
@@ -4,20 +4,23 @@ defmodule V3Api.HeadersTest do
   use ExUnit.Case, async: true
 
   test "always adds api header" do
-    assert Headers.build("API_KEY", use_cache?: false) == [
```

Comment:
It looks like the env var must have been set differently (possibly via config?) in CI before if this was previously passing. Doesn't seem important to me, though.